src/errorCodes.js

```diff
@@ -0,0 +1,56 @@
+import { Platform } from 'react-native'
```

Comment:
we do not use `Platform` here as I see

Reply:
https://github.com/tipsi/tipsi-stripe/blob/533d4773896868f871227d1af1c50e38ca9beee2/src/errorCodes.js#L36

Reply:
OOK